### PR TITLE
Auth With Chahub

### PR DESCRIPTION
**Pull request by Tthomas63**
- Bump Social Auth versions for core and the django specific app
- Add missing abstract method to OAuth Backend for Chahub

### CODE REVIEW

File: src/templates/registration/login.html [hunk 1]
```diff
@@ -37,6 +37,7 @@ <h2 class="ui blue header">
                     </div>
                     <button class="ui blue submit button" type="submit">Login</button>
                     <div class="ui divider"></div>
+                    <a href="{% url "social:begin" "codalab" %}">Chahub SIGN IN</a>
```
Comment (ckcollab) [inline src/templates/registration/login.html]: please change to "chahub"

File: src/utils/oauth_backends.py [hunk 2]
```diff
@@ -1,24 +1,30 @@
...
 
+BASE_URL = settings.SOCIAL_AUTH_CHAHUB_BASE_URL
 
-BASE_URL = "https://competitions-v2-staging-pr-17.herokuapp.com"
 
-
-class CodalabOAuth2(BaseOAuth2):
+class ChahubOAuth2(BaseOAuth2):
     """Github OAuth authentication backend"""
     name = 'codalab'
```
Comment (ckcollab) [inline src/utils/oauth_backends.py]: "Chahub" here

File: src/utils/oauth_backends.py [hunk 3]
```diff
@@ -1,24 +1,30 @@
...
+    def get_user_id(self, details, response):
+        return details.get('id')
 
     def get_user_details(self, response):
         access_token = response['access_token']
         my_profile_url = "{}my_profile/".format(self.API_URL)
         data = self.get_json(my_profile_url, headers={'Authorization': 'Bearer {}'.format(access_token)})
-        print("usa details", data)
+
+        print("user details", data)
```
Comment (ckcollab) [inline src/utils/oauth_backends.py]: remove print

**Comment by jimmykodes** [comment]
@Tthomas63 Is this still a relevant branch/PR? or have these changes been included elsewhere?

**Pull request closed without merging**